larq_compute_engine/core/bgemm_trmul_params.h

```diff
@@ -35,21 +35,20 @@ void PopulateBinaryTrMulParams(ruy::TrMulParams* params) {
     return;
   }
 
-  using PackedTBitpacked = ruy::PackedType<ThePath, TBitpacked>;
   using Kernel = BgemmKernel<ThePath, DstScalar, MulParamsType>;
   using LhsKernelLayout = typename Kernel::LhsLayout;
   using RhsKernelLayout = typename Kernel::RhsLayout;
 
   params->path = ThePath;
 
-  CreatePackedMatrix<TBitpacked, PackedTBitpacked>(
+  CreatePackedMatrix<TBitpacked, TBitpacked>(
       Side::kLhs, ToKernelLayout<LhsKernelLayout>(), params);
-  CreatePackedMatrix<TBitpacked, PackedTBitpacked>(
+  CreatePackedMatrix<TBitpacked, TBitpacked>(
       Side::kRhs, ToKernelLayout<RhsKernelLayout>(), params);
   params->run_pack[Side::kLhs] =
-      &RunPack<ThePath, LhsKernelLayout, TBitpacked, PackedTBitpacked>;
+      &compute_engine::tflite::RunPack<ThePath, LhsKernelLayout>;
```

Comment:
From what I understand from the comments in `ruy_pack.h` , we only implement optimized 4x4 layouts (required for the RHS) and not 8x4 layouts for the LHS, since the LHS is cached anyway; makes sense :+1:. However here we call `compute_engine::tflite::RunPack` on the LHS, and I don't understand why that would not throw a compiler error because that `RunPack` only seems to have support for 4x4, and I don't see any code that makes it fallback to the generic Ruy packing code.